documentation/contributing/index.markdown

```diff
@@ -100,9 +94,9 @@ Dave Coleman | Correll Lab, CU Boulder | [davetcoleman](https://github.com/davet
 Robert Haschke | CITEC, Bielefeld University | [rhaschke](https://github.com/rhaschke) | Fixing Bugs, Reviewing PRs
 Michael Görner | University of Hamburg | [v4hn](https://github.com/v4hn) | Fixing Bugs, New Features
 Isaac IY Saito | Tokyo Open Source Robotics | [130s](https://github.com/130s) | Releasing and Continuous Integration
-Sachin Chitta | Kinema Systems | [sachinchitta](https://github.com/sachinchitta) | Original Project Manager and Author of MoveIt!
+Sachin Chitta | Kinema Systems | [sachinchitta](https://github.com/sachinchitta) | Original Manager & Author of MoveIt!
 Ioan Sucan | Google X | [isucan](https://github.com/isucan) | Original Author of MoveIt!
 Ian McMahon | Rethink Robotics | [IanTheEngineer](https://github.com/IanTheEngineer) | Releasing
-Gijs Hoorn | Delft University / ROS-Industrial | [gavanderhoorn](https://github.com/gavanderhoorn) | IKFast Plugin
+Gijs Hoorn | Delft Univ / ROS-Industrial | [gavanderhoorn](https://github.com/gavanderhoorn) | IKFast Plugin
```

Comment:
Somewhat late, but it's actually "Delft University of Technology" unfortunately.